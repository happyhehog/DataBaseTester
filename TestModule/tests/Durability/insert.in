[@Clear]
[@PrintLevel] MAIN

@@ =========================
@@ Testing DURABILITY insert
@@ =========================

[@NoOutput]
CREATE TABLE t(id INT, height FLOAT, nickname CHAR(20));
INSERT INTO t VALUES (1, 150.0, 'hedgehog');
INSERT INTO t VALUES (2, 120.0, 'linux');
[@NoOutput]

[@Print] Before restart
SELECT * FROM t;

[@RestartServer] 5 2

[@NoOutput]
BEGIN;
[#Repeat] 200
INSERT INTO t VALUES($i, $i, '$i');
COMMIT;
[@NoOutput]

[@Print] \nAfter restart
SELECT * FROM t;