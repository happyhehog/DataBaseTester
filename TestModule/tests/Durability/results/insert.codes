{"status":"OK","command":"SELECT * FROM t;","answer":"id|height    |nickname  |\n1 |150.000000|'hedgehog'|\n2 |120.000000|'linux'   |"}
{"status":"OK","command":"SELECT * FROM t;","answer":"id|height    |nickname  |\n1 |150.000000|'hedgehog'|\n2 |120.000000|'linux'   |"}
