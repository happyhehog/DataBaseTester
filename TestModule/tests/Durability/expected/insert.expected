Before restart
id|height    |nickname  |
1 |150.000000|'hedgehog'|
2 |120.000000|'linux'   |

After restart
id|height    |nickname  |
1 |150.000000|'hedgehog'|
2 |120.000000|'linux'   |
